### PR TITLE
LPS-164071 Update empty state title for folder (not Home)

### PR DESCRIPTION
**Pull request by marcogalluzzi**
Added the conditional operator to check for the parent resource primary key. If nothing exists the message is 'knowledge-base-is-empty', if we are inside an empty folder the already existing message 'this-folder-is-empty' is used.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view.jsp [hunk 1]
```diff
@@ -326,7 +326,7 @@ KBArticleURLHelper kbArticleURLHelper = new KBArticleURLHelper(renderRequest, re
 						actionDropdownItems="<%= kbAdminManagementToolbarDisplayContext.getEmptyStateActionDropdownItems() %>"
 						animationType="<%= EmptyResultMessageKeys.AnimationType.EMPTY %>"
 						buttonCssClass="secondary"
-						title='<%= LanguageUtil.get(request, "knowledge-base-is-empty") %>'
+						title='<%= (parentResourcePrimKey == 0) ? LanguageUtil.get(request, "knowledge-base-is-empty") : LanguageUtil.get(request, "this-folder-is-empty") %>'
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view.jsp:329]: it's better to use `KBFolderConstants.DEFAULT_PARENT_FOLDER_ID` than `0`

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view.jsp (hunk 1 above)
Comment (marcogalluzzi) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view.jsp:329]: That's true!

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**